iml-agent/src/action_plugins/stratagem/action_purge.rs

```diff
@@ -48,10 +48,10 @@ pub fn read_mailbox(
                     .last()
                     .map(String::from)
             })
-            .chunks(10) // @@ Get number from liblustreapi
+            .chunks(llapi.rmfids_size())
             .for_each(move |fids| {
                 tokio::spawn(
-                    rm_fids(mntpt.clone(), fids)
+                    rm_fids(llapi.clone(), fids)
```

Comment:
Nit: using Arc::clone(&llapi) makes it more clear that this just creating a new reference to the pointer and is thus not expensive.